snapshotcreation/snapshotcreation.py

```diff
@@ -24,36 +36,32 @@ def lambda_handler(event, context):
 
     # Iterate through instances identified by the filter.
     for instance in ec2.instances.filter(Filters=event['Filters']):
+        instance_name = instance.instance_id
+        instance_tags = []
+
+        # If a Name tag is available, use it to identify the instance
+        # instead of the instance_id.
+        for tag in instance.tags:
+            if tag['Key'] == 'Name' and tag['Value'] != '':
+                instance_name = tag['Value']
+            else:
+                instance_tags.append(tag)
+
+        try:
+            # Create the AMI
+            image_name = instance_name + '-' + get_unix_timestamp()
```

Comment:
It appears that the AMI creation steps we're currently performing in Ansible are using Unix timestamps. Using them also doesn't create much of a hinderance as AMIs contain easily readable dates in their metadata anyway and those fields are typically displayed/sortable by default in the AWS console. The Unix timestamp is acting more as a unique identifier in this case.